test.py

```diff
@@ -0,0 +1,59 @@
+import time
+import os
+import sys
+import pytest
+import subprocess
+
+# Add the project root directory to the system path
+root_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+sys.path.append(root_dir)
+
+from client.client import Client
+
+TEST_PPT_PATH = "/Users/damiantissembaum/Documents/year_3/TEMP/Tests-3.pptx"
```

Comment:
That makes your program hard to run.
If you'd put your test.pptx inside your project folder then anybody could run your code.